front/components/poke/PokeNavbar.tsx

```diff
@@ -24,7 +33,71 @@ export const PokeNavbar: React.FC = () => (
         </Link>
       </div>
     </div>
+    <PokeSearchCommand />
   </nav>
 );
 
 export default PokeNavbar;
+
+export function PokeSearchCommand() {
+  const [open, setOpen] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const { isError, isLoading, results } = usePokeSearch({
+    // Disable search until the user has typed at least 2 characters.
+    disabled: searchTerm.length < 2,
+    search: searchTerm,
+  });
+
+  useEffect(() => {
+    const down = (e: KeyboardEvent) => {
+      if (e.key === "k" && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        setOpen((open) => !open);
+      }
+    };
+    document.addEventListener("keydown", down);
+    return () => document.removeEventListener("keydown", down);
+  }, []);
+
+  return (
+    <>
+      <PokeButton variant="outline" size="sm" onClick={() => setOpen(true)}>
+        Search
+      </PokeButton>
+      <PokeCommandDialog
+        open={open}
+        onOpenChange={setOpen}
+        className="bg-structure-50 sm:max-w-[600px]"
+        shouldFilter={false}
+      >
+        <PokeCommandInput
+          placeholder="Type a command or search..."
+          onValueChange={(value) => setSearchTerm(value)}
+          className="border-none focus:outline-none focus:ring-0"
+        />
+        <PokeCommandList>
+          {isLoading && <div className="p-4 text-sm">Searching...</div>}
+          {!isError && !isLoading && results.length === 0 && (
```

Comment:
nit && searchTerms && searchTerms.length >= 2

and a special / no message when nothing has been searched